### DISCUSSION TITLE
Symbol- and Line-Completion

### DISCUSSION THREAD

**Discussion by Dentarthurdent42** [community/community#38875] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Copilot is amazing, *however*, sometimes it gets the first few symbols or lines right, while the rest is repetitive or incorrect. It would be extremely welcome to be able to only complete one symbol or line at a time. As it stands, if Copilot suggests 12 lines of code and I only need the first like 3, I have to either type out those 3 lines or complete all 12 and delete the last 9. If there could be some shortcut like Alt-Tab or Ctrl-Tab or something, that would just be the cat's pajamas.

**Comment by iamcjwilliams** [comment]
@Dentarthurdent42 Thank you for your feedback, which IDE are you using?

**Reply by Dentarthurdent42** [reply]
I use VSCode! Sorry, shoulda mentioned that.

**Reply by iamcjwilliams** [reply]
@Dentarthurdent42 Our copilot team is investigating this issue

**Comment by cmp-nct** [comment]
It's nice how you try to be polite and use the word "sometimes" :-)

If you use Jetbrains IDE, I've solved the problem here: https://github.com/cmp-nct/Stewardess
It still has a few bugs but does what you need using a secondary shortcut. 
You can bind Ctrl-Tab but that won't work on Windows of course.

**Reply by Dentarthurdent42** [reply]
I'm actually looking into JetBrains atm. I'll definitely check out your solution if I end up getting it, but while money is tight, I'm stuck with VSCode 😐
